### PR TITLE
Add bookmarked snackbar and back button to detail view.

### PR DESCRIPTION
**Pull request by cbonoz**

### CODE REVIEW

File: Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:tools="http://schemas.android.com/tools" android:id="@+id/agendaDetailView"
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml]: should be on separate lines?

File: Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml [hunk 2]
```diff
@@ -17,6 +17,16 @@
             android:layout_marginBottom="16dp"
             android:orientation="horizontal">
 
+            <ImageButton
+                    android:id="@+id/image_back"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:padding="8dp"
+                    android:layout_gravity="center_vertical"
+                    android:contentDescription="@string/hint_bookmark"
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml]: probably needs new hint text instead of telling users this is a "bookmark" :-)

Comment (cbuonocore-toasttab) [reply Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml]: updated

File: Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml (hunk 1 above)
Comment (cbuonocore-toasttab) [reply Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml]: reformatted

**Pull request merged**